apps/docs/src/guide/errors/index.md

```diff
@@ -1,3 +1,270 @@
+
+Ensure that the value you've supplied to the big number is compatible with the value you are converting to.
+
+### `DUPLICATED_POLICY`
+
+When there are more than policies with the same type, for a transaction.
+
+Ensure that there are no duplicate (by type) policies for a transaction.
+
+### `ERROR_BUILDING_BLOCK_EXPLORER_URL`
+
+When more than one of the following options is passed: `path`, `address`, `txId`, `blockNumber`.
+
+Check that only one of the following options is passed: `path`, `address`, `txId`, `blockNumber`.
+
+### `FUNCTION_NOT_FOUND`
+
+When the function with the given name, signature or selector is not found in the ABI.
+
+Check that the function name, signature or selector is correct and exits on the ABI.
+
+### `GAS_LIMIT_TOO_LOW`
+
+When the gas limit is lower than the minimum gas limit.
+
+Increase the gas limit to be greater than the minimum gas limit.
+
+### `GAS_PRICE_TOO_LOW`
+
+When the gas price is lower than the minimum gas price.
+
+Increase the gas price to be greater than the minimum gas price.
+
+### `HD_WALLET_ERROR`
+
+A hardware wallet will throw for unsupported configurations.
+
+The error message will determine which element of the configuration is incorrect. It could be due to the public or private key or when configuring to/from an extended key.
+
+### `INVALID_CHECKSUM`
+
+Checksum validation failed for the provided mnemonic.
+
+Ensure that the mnemonic is correct.
+
+### `INVALID_CONFIGURABLE_CONSTANTS`
+
+When the program type either: does _not_ have configurable constants to be set; or the provided configurable constant does not belong to the program type, as defined by its ABI.
+
+Ensure the configurable constants provided are correct and are defined in ABI.
+
+### `INVALID_COMPONENT`
+
+When an expected component is not found in the ABI or is malformed.
+
+Ensure that you have correctly formed Sway types for [Arrays](../types/arrays.md) and [Vectors](../types/vectors.md).
+
+### `INVALID_CREDENTIALS`
+
+When the password provided is incorrect.
+
+Ensure that the password is correct.
+
+### `INVALID_DATA`
+
+When the value is more than the max value.
+
+Ensure that the value is less than the max value.
+
+### `INVALID_ENTROPY`
+
+When the entropy is not: between 16 and 32 bytes; a multiple of 4.
+
+Ensure that the entropy is between 16 and 32 bytes and a multiple of 4.
+
+### `INVALID_EVM_ADDRESS`
+
+When the provided EVM address is invalid.
+
+Ensure that the [EVM address](../types/evm-address.md) is valid.
+
+### `INVALID_INPUT_PARAMETERS`
+
+When the provided input parameters are _not_ valid.
+
+The error message will determine which parameter is missing. It could be that the provided program type is not one of the following `contract`, `script`, or `predicate`.
+
+### `INVALID_MNEMONIC`
+
+When the supplied mnemonic is invalid.
+
+Check the message for more details. It could be that the mnemonic phrase word length is \_not\_\_ one of the following: 12, 15, 18, 21, or 24 lengths.
```

Comment:
```suggestion
Check the message for more details. It could be that the mnemonic phrase word length is _not_ one of the following: 12, 15, 18, 21, or 24 lengths.
```